search-commons/src/test/resources/datasource_urls_resource.json

```diff
@@ -92,9 +94,10 @@
   "JOURNAL=1879-1840": 1,
   "JOURNAL=0003-4339": 1,
   "JOURNAL=Annales+des+Sciences": 1,
-  "LICENSE=CC-BY": 5,
-  "LICENSE_NOT=CC-BY": 16,
-  "LICENSE_SHOULD=https://creativecommons.org/licenses/by/4.0": 5,
+  "LICENSE=CC-BY": 11,
+  "LICENSE_NOT=CC-BY": 10,
+  "LICENSE=https://creativecommons.org/licenses/by/4.0": 11,
```

Comment:
Do we support search for older versions as well? That is, versions 1.0, 2.0, 2.5 and 3.0??

Reply:
yes